dbms/src/Storages/Transaction/DatumCodec.cpp

```diff
@@ -148,21 +157,21 @@ void SkipVarInt(size_t & cursor, const String & raw_value)
     SkipVarUInt(cursor, raw_value);
 }
 
-inline Int8 getWords(PrecType prec, ScaleType scale)
+inline Int32 getWords(PrecType prec, ScaleType scale)
```

Comment:
why we need change Int8 to Int32?

Reply:
Since the return value is assigned to 'int' type value, and assign signed char to int lead to tidy clang error. It suggests to convert char to unsigned char first, while it will cause incorrect result. Thus, I changed the getWords signature.

Reply:
ok.